homework/homework10/return_number.py

```diff
@@ -0,0 +1,27 @@
+"""Return the number.
+This program checks if the result of the function is a number
+and prints an error message if it is not.
+"""
+import math
+
+
+def is_number(func):
+    """ A decorator that checks if the result of the function is a number"""
+    def wrapper(*args, **kwargs):
+        result = func(*args, **kwargs)
+        if not isinstance(result, (int, float)):
+            print('Error! The result is not a number.')
+        return result
+    return wrapper
+
+
+@is_number
+def summ(*args, **kwargs):
+    """Returns the sum of all arguments if is number"""
+    return math.fsum(args) + math.fsum(kwargs.values())
+
```

Comment:
Here, I just wanted to use the math module. I agree, it is better to use just sum because in it the type of the return value depends on the numbers being added. Fixed